src/gameplaykit.cs

```diff
@@ -48,7 +77,7 @@ interface GKAgent {
 		float Radius { get; set; }
 
 		[Export ("speed")]
-		float Speed { get; }
+		float Speed { get; [iOS (10,0), TV(10,0), Mac (10,12, onlyOn64: true)] set; }
```

Comment:
It's one for source compatibility (less a problem, it's an easy fix). Not sure for a binary... worth a test.